source/includes/rate-limit/_root.md

```diff
@@ -0,0 +1,31 @@
+# Rate Limiting
+
+
+For our content and services to be available to everyone we have to make sure
+that our resources are being distributed fairly. 
+Due to the above we have implemented a rate limiter for our service which
+limits the requests that users can perform in a time window.
+Depending of the complexity of each request different endpoints can have
+different limits.
+
+Static pages in principle will have a very high limit.
```

Comment:
I think we don't need to mention anything about static pages since most people are probably not aware of them.